includes/payment-methods/class-wc-stripe-upe-payment-method.php

```diff
@@ -272,12 +272,24 @@ public function get_supported_currencies() {
 	/**
 	 * Returns the HTML for the subtext messaging in the old settings UI.
 	 *
+	 * @param string $stripe_method_status (optional) Status of this payment method based on the Stripe's account capabilities
 	 * @return string
 	 */
-	public function get_subtext_messages() {
+	public function get_subtext_messages( $stripe_method_status ) {
 		// can be either a `currency` or `activation` messaging, to be displayed in the old settings UI.
 		$messages = [];
 
+		if ( ! empty( $stripe_method_status ) && 'active' !== $stripe_method_status ) {
+			$text            = __( 'Pending activation', 'woocommerce-gateway-stripe' );
+			$tooltip_content = sprintf(
+				/* translators: %1: Payment method name %2: Store admin e-mail. */
+				esc_attr__( '%1$s won\'t be visible to your customers until you provide the required information. Follow the instructions we\'ve sent to %2$s.', 'woocommerce-gateway-stripe' ),
+				$this->get_label(),
+				get_option( 'admin_email' )
```

Comment:
Should this be the Stripe account email (rather than the site's admin email)?

Reply:
After some research, unfortunately I found out this is not possible. Stripe API does not retrieve that information unless we set it manually as a metadata. What about changing the wording to:

>Follow the instructions Stripe has sent to your e-mail.

Reply:
I think that's perfect!